types/simple-react-clipboard/index.d.ts

```diff
@@ -0,0 +1,19 @@
+import { ReactNode } from "react";
+
+interface ClipboardProps {
+    render: () => ReactNode;
+    text: string;
+    props: object;
```

Comment:
```suggestion
    props?: object | undefined;
```
is this one optional as the others?
https://github.com/donfour/simple-react-clipboard/blob/master/src/lib/index.js#L64

Reply:
Yes, it should be optional. Nice catch ⭐